pages/whats-new/2020-11-17-batch-imports-and-mailchimp-source.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: "Batch Imports and Mailchimp Source"
+date: "2020-11-17"
+tags: ["new", "improvement", "fix"]
+description: "Batch Imports and Mailchimp as a Source"
+github: "https://github.com/grouparoo/grouparoo/releases/tag/v0.1.24"
```

Comment:
Got the URL right! https://github.com/grouparoo/grouparoo/releases/tag/v0.1.24

Reply:
Predicted! :D